c7n/resources/route53.py

```diff
@@ -616,6 +618,9 @@ def process(self, resources, event=None):
         return results
 
 
+region = 'us-west-2'
```

Comment:
can we Capitalize this so it stands out as a constant a little more clearly as a global versus a local variable

Reply:
Updated variable name. Thanks.